ecosystem/sep-0006.md

```diff
@@ -987,7 +987,8 @@ Name | Type | Description
 `stellar_transaction_id` | string | (optional) transaction_id on Stellar network of the transfer that either completed the deposit or started the withdrawal.
 `external_transaction_id` | string | (optional) ID of transaction on external network that either started the deposit or completed the withdrawal.
 `message` | string | (optional) Human readable explanation of transaction status, if needed.
-`refunded` | boolean | (optional) Should be true if the transaction was refunded. Not including this field means the transaction was not refunded.
+`refunded` | boolean | (**deprecated**, optional) This field is deprecated in favor of the `refund` object. Should be true if the transaction was refunded. Not including this field means the transaction was not refunded.
```

Comment:
Even if we are deprecating this field, I think we need to define for this field if it is true or false for partial refunds. Looking at some other APIs for inspiration, like Stripe's Charge API, I'd be inclined to say it should be false until the transaction is fully refunded.

On that note, it is unclear why we would deprecate this field. Is there significant value in us doing so? Since we are going to support the field indefinitely it might make the API less confusing to leave it not-deprecated.

Reply:
>  I'd be inclined to say it should be false until the transaction is fully refunded.

That makes sense to me.

> it is unclear why we would deprecate this field. Is there significant value in us doing so?

I'm don't think the value is significant, but I think it makes more sense to have all refund-related info in a single object.

Reply:
Cool! In that case, can we update the parameter description to explicitly say it's `true` for a full refund?